project-base/storefront/components/Forms/Button/Button.tsx

```diff
@@ -1,10 +1,9 @@
 import { ButtonHTMLAttributes, forwardRef } from 'react';
-import { useFormContext } from 'react-hook-form';
 import { twMergeCustom } from 'helpers/twMerge';
 
 type NativeButtonProps = Omit<ButtonHTMLAttributes<HTMLButtonElement>, 'type' | 'disabled'>;
 
-type Props = NativeButtonProps &
+export type ButtonProps = NativeButtonProps &
     Omit<NativeButtonProps, 'type' | 'disabled'> & {
         type?: 'button' | 'submit' | 'reset';
```

Comment:
Since type "submit" is used in one place and type "reset" in none, I am thinking if there maybe is a way to remove this prop from the Button component :thinking:

Reply:
Hmm, I guess the type is really not necessary. Good idea. Refactored.